tests_speculos/test-basic-slot-status-bad-net.js

```diff
@@ -24,15 +25,13 @@ common.curlScreenShot(scriptName); console.log(common.humanTime() + " // screen
 common.testStep(" - - -", "await app.setSlot() // expectedSlot=" + expectedSlot + " expectedAccount=" + expectedAccount + " expectedPath=" + expectedPath);
 const setSlotResponse = await app.setSlot(expectedSlot, expectedAccount, expectedPath);
 assert.equal(setSlotResponse.returnCode, 0x6984);
-assert.equal(common.mockTransport.hexApduCommandOut.length, 1)
-assert.equal(common.mockTransport.hexApduCommandIn.length, 1)
+assert.equal(transport.hexApduCommandOut.length, 1)
+assert.equal(transport.hexApduCommandIn.length, 0)
```

Comment:
Why the value change here?

Reply:
As Ledger's Transport throws an exception, it is not possible to look at outgoing APDU in our spy transport.